src/core/net/tcp6.hpp

```diff
@@ -381,14 +397,15 @@ class Tcp : public InstanceLocator, private NonCopyable
          *
          * @retval kErrorNone    Successfully stopped listening on the TCP listener.
          * @retval kErrorFailed  Failed to stop listening on the TCP listener.
+         *
          */
         Error StopListening(void);
 
         /**
          * Deinitializes this TCP listener.
          *
          * This means that OpenThread no longer keeps track of this TCP listener and
-         * deallocates all resources it has internally allocated for this TCP listener.
+         * de-allocates all resources it has internally allocated for this TCP listener.
```

Comment:
I believe `deallocates` is accepted as a word: https://www.google.com/search?q=deallocates
```suggestion
         * deallocates all resources it has internally allocated for this TCP endpoint.
```